src/app/components/audio-recordings/pages/list/list.component.spec.ts

```diff
@@ -1,24 +1,37 @@
-import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { MockBawApiModule } from "@baw-api/baw-apiMock.module";
+import { AudioRecording } from "@models/AudioRecording";
+import { createRoutingFactory, SpectatorRouting } from "@ngneat/spectator";
+import { SharedModule } from "@shared/shared.module";
+import { DateTime } from "luxon";
 import { AudioRecordingsListComponent } from "./list.component";
 
-// TODO
-xdescribe("AudioRecordingsListComponent", () => {
-  let component: AudioRecordingsListComponent;
-  let fixture: ComponentFixture<AudioRecordingsListComponent>;
+describe("AudioRecordingsListComponent", () => {
+  let spectator: SpectatorRouting<AudioRecordingsListComponent>;
 
-  beforeEach(async () => {
-    await TestBed.configureTestingModule({
-      declarations: [AudioRecordingsListComponent],
-    }).compileComponents();
+  const createComponent = createRoutingFactory({
+    component: AudioRecordingsListComponent,
+    imports: [MockBawApiModule, SharedModule],
   });
 
-  beforeEach(() => {
-    fixture = TestBed.createComponent(AudioRecordingsListComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
-  });
+  function setup() {
+    spectator = createComponent({ detectChanges: false });
+    return spectator;
+  }
+
+  beforeEach(() => setup());
 
   it("should create", () => {
-    expect(component).toBeTruthy();
+    expect(spectator.component).toBeInstanceOf(AudioRecordingsListComponent);
+  });
+
+  it("should format dates correctly", () => {
+    const mockDateTime = DateTime.fromJSDate(
+      new Date("1995-12-17T03:24:00")
+    );
+    const mockAudioRecording = new AudioRecording({ recordedDate: mockDateTime});
+
+    const expectedDateTime = "3:24 AM, Sunday, 17, December 1995";
+
+    expect(spectator.component.getRecordingDate(mockAudioRecording)).toEqual(expectedDateTime);
```

Comment:
this test does not actually test for the problem described in #1997 - that a UTC date from the server is shown in the user's local time zone.

Reply:
Ok, so the scope of this ticket is to not show dates provided by the server in the users timezone